packages/blade/.storybook/react/preview.js

```diff
@@ -31,28 +32,30 @@ export const parameters = {
 export const decorators = [
   (Story, context) => {
     const getThemeTokens = () => {
-      if (context.globals.themeTokens === 'paymentTheme') {
+      if (context.globals.themeTokenName === 'paymentTheme') {
         return paymentTheme;
       }
-      if (context.globals.themeTokens === 'bankingTheme') {
+      if (context.globals.themeTokenName === 'bankingTheme') {
         return bankingTheme;
       }
     };
-
     return (
-      <BladeProvider
-        key={`${context.globals.themeTokens}-${context.globals.colorScheme}`}
-        themeTokens={getThemeTokens()}
-        colorScheme={context.globals.colorScheme}
-      >
-        <Story />
-      </BladeProvider>
+      <>
+        <GlobalStyle />
```

Comment:
need this to inject storybook's `css` resets as I'm using their native components in our docs later.